torch/csrc/jit/codegen/cuda/scheduler/reduction_utils.h

```diff
@@ -22,7 +22,7 @@ TensorView* scheduleReductionTV(
     bool has_iter_axis);
 
 // Inlining function intended for single or multi reduction fusions.
-void multiReductionInliner(
+TORCH_CUDA_CU_API void multiReductionInliner(
```

Comment:
We need to expose this because they are used in benchmarks?!

Reply:
nvm... I think tests are also built separately so these should be needed as well.
hmmm.. but how did we past upstream CI :confused:

Reply:
Some of these routines are used by outer-reduction tests that are not yet merged into devel.